app/Console/Kernel.php

```diff
@@ -43,6 +44,8 @@ protected function schedule(Schedule $schedule)
 
         // Schedule site stat updates for each wiki and platform-summary
         $schedule->command('schedule:stats')->daily();
+
+        $schedule->job(new PollForMediaWikiJobsJob)->everyMinute();
```

Comment:
As Laravel is using cron schedules for defining job rates, there is no way to have this run more often. Unless we roll our own ticker or similar. Not sure if this frequency is ok for us?

Reply:
I'm definitely *slightly* worried that this is too slow; maybe it is ok as a first solution though. We could also leave the "running stuff on requests" enabled as well to mitigate this?

I would prefer 1/s up to 1/10s since I think a minute is long enough user would notice the job not having been done (e.g. they start immediately searching for the thing they just did)

Reply:
Wouldn't that require its own process as it's supposed to run forever?